### DISCUSSION TITLE
Controlling Fury windows by HTC VIVE

### DISCUSSION THREAD

**Discussion by alecrimi** [fury-gl/fury#668] [General]
**Is your feature request related to a problem? Please describe.**
Given the fact that Fury is based on VTK should be possible to link HTC controllers for VR, but how?

**Comment by skoudoro** [comment]
Hi @alecrimi,

- You need to compile VTK with this option and install your compiled version. 
- By default after compiling VTK with openvr option,  the object factory mechanism should replace the core rendering classes vtkRenderWindow by vtkRenderingOpenVR, it is transparent for you, and it should work.
- concerning interaction, you might have some work to do. only basic interactions are working.
- Concerning switching dynamically between VR mode and Normal mode, it is not done yet.

I hope this helps and let us know if you have other questions or if you want to meet online.
